Util/ClassUtils.php

```diff
@@ -22,10 +20,25 @@ private function __construct()
      */
     public static function getClass($object): string
     {
-        if (\class_exists(DoctrineClassUtils::class)) {
-            return DoctrineClassUtils::getClass($object);
+        $className = \get_class($object);
+
+        // __CG__: Doctrine Common Marker for Proxy (ODM < 2.0 and ORM < 3.0)
```

Comment:
I think we should credit original source here

Reply:
@garak ofc